examples/pubsub/tutorial_pubsub_subscribe.c

```diff
@@ -93,11 +93,12 @@ addDataSetReader(UA_Server *server) {
     /* The following parameters are used to show that the data published by
      * tutorial_pubsub_publish.c is being subscribed and is being updated in
      * the information model */
-    UA_UInt16 publisherIdentifier = 2234;
-    readerConfig.publisherId.type = &UA_TYPES[UA_TYPES_UINT16];
-    readerConfig.publisherId.data = &publisherIdentifier;
-    readerConfig.writerGroupId    = 100;
-    readerConfig.dataSetWriterId  = 62541;
+    UA_UInt16 publisherIdentifier      = 2234;
+    readerConfig.publisherId.type      = &UA_TYPES[UA_TYPES_UINT16];
+    readerConfig.publisherId.data      = &publisherIdentifier;
+    readerConfig.writerGroupId         = 100;
+    readerConfig.dataSetWriterId       = 62541;
+    readerConfig.messageReceiveTimeout = 200.0; /* Timeout must be greater than publishing interval of corresponding WriterGroup */
```

Comment:
Yes, because it's only used for PubSub component timeout handling -> DataSetReader MessageReceiveTimeout
If this feature is enabled the parameter must be set, otherwise the timeout occurs immediately